src/Microsoft.SqlTools.ServiceLayer/EditData/SmoEditMetadataFactory.cs

```diff
@@ -86,6 +86,10 @@ public EditTableMetadata GetObjectMetadata(DbConnection connection, string[] obj
                     throw new ArgumentOutOfRangeException(nameof(objectType), SR.EditDataUnsupportedObjectType(objectType));
             }
 
+            // Filter out dropped ledger columns from the list of columns to be returned
+            //
+            smoResult.Columns.ClearAndInitialize("[(@IsDroppedLedgerColumn=0)]", new [] {nameof(Column.IsDroppedLedgerColumn), nameof(Column.DataType)});
+
             // A bug in SMO makes it necessary to call refresh to attain certain properties (such as IsMemoryOptimized)
```

Comment:
>  // A bug in SMO [](http://example.com/codeflow?start=10&length=17)

what bug is that?

Reply:
Yeah, I'm not sure what that's referring to.